osquery/events/linux/auditeventpublisher.cpp

```diff
@@ -25,10 +26,14 @@ DECLARE_bool(audit_allow_sockets);
 DECLARE_bool(audit_allow_user_events);
 DECLARE_bool(audit_allow_selinux_events);
 DECLARE_bool(audit_allow_kill_process_events);
+DECLARE_bool(audit_allow_apparmor_events);
 
 REGISTER(AuditEventPublisher, "event_publisher", "auditeventpublisher");
 
 namespace {
+
+const std::string kAppArmorEventMarker{"apparmor"};
```

Comment:
Are you sure this should be "just `apparmor`" and not `apparmor=`?

Reply:
Double checked this place and that's fine. Below you can find the events processing function and it processes events that have already been tokenized, so it's not about looking up a substring in a raw string, it's about checking the presence of "apparmor" key in the AuditEventRecord internal map.